libcpuid/exports.def

```diff
@@ -30,3 +30,5 @@ cpu_clock_by_ic @26
 cpuid_get_total_cpus @27
 cpu_msr_driver_open_core @28
 cpuid_get_vendor @29
+cpu_rdmsr_range @30
+get_bits_value @31

```

Comment:
For a basic usage, I think a developer will only use `cpu_rdmsr_range()`.
The `get_bits_value()` can avoid to read the same register several times.
But we can ride off `get_bits_value()`, it can be easily removed from `cpu_msrinfo()`.